factcast-core/src/main/java/org/factcast/core/util/FactCastJson.java

```diff
@@ -158,10 +177,21 @@ public static JsonNode toJsonNode(Map<String, Object> jsonAsMap) {
     @SneakyThrows
     public static <A> A readValueFromBytes(Class<A> type, byte[] bytes) {
         return objectMapper.readerFor(type).readValue(bytes);
-
     }
 
     public static ObjectMapper mapper() {
         return objectMapper;
     }
+
+    @SneakyThrows
+    public static long calculateHash(Class<?> projectionClass) {
```

Comment:
this uses the FCJson internal objectmapper, but we agreed that people must be able to swap the obejctmapper used for eventserialization. therefore i'd suggest to move this out of here and into the eventserializer ?